docs/application/dotnet/guides/nui/text.md

```diff
@@ -24,15 +24,15 @@ The `TextLabel` class is lightweight, non-editable, and do not respond to user i
 
 To create a text label:
 
-1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+1.  Create an instance of the `TextLabel` class and define the label text as a parameter. To make text visible, set `TextColor` property to `Color.White`:
 
     ```csharp
-    Window window = Window.Instance;
-    TextLabel label = new TextLabel("Hello World");
-    window.Add(label);
+    TexLabel label = new TextLabel("Hello World");

```

Comment:
It seems "t" is missing. Would you please check?

```suggestion
    TextLabel label = new TextLabel("Hello World");
```

Reply:
Done